.circleci/config.yml

```
@@ -36,12 +39,14 @@ jobs:
             # fallback to using the latest cache if no exact match is found
             - v1-dependencies-
 
-      - run: cat /dev/null | sbt 'Test / compile'
+      # Compile, run tests, and produce coverage reports!
+      - run: cat /dev/null | sbt clean coverage test coverageReport coverageAggregate shutdown
```

Or you could redirect in an empty input:
```suggestion
      - run: sbt clean coverage test coverageReport coverageAggregate shutdown << /dev/null
```

Either one of these would stop another shell process from being started.